src/zui/l10n/messageIds.ts

```diff
@@ -24,6 +24,36 @@ export default makeMessages('zui', {
     copiedValue: m<{ value: string }>('Copied "{value}"'),
     copy: m('Copy'),
   },
+  createPerson: {
+    cancel: m('Cancel'),
+    createBtn: m('Create'),
+    defaultitle: m('Create person'),
+    genders: {
+      f: m('Female'),
+      m: m('Male'),
+      o: m('Other'),
+      unknown: m('Unknown'),
+    },
+    showAllFields: m('Show all fields'),
+    submitLabel: {
+      add: m('Create & add'),
+      assign: m('Create & assign'),
+      default: m('Create'),
+    },
+    title: {
+      addTo: m<{ value: string }>('and add to {value}'),
+      assignTo: m<{ value: string }>('and assign to {value}'),
```

Comment:
I think the names of these messages could reflect that these messages are used for journeys. Maybe:

```suggestion
      addToJourney: m<{ journey: string }>('and add to {journey}'),
      assignToJourney: m<{ value: string }>('and assign to {journey}'),
```

If you are also using these for something else, please create separate messages. Some languages will want to treat those strings differently.